osflow/Makefile

```diff
@@ -94,6 +94,14 @@ iCEBreaker:
 	  NEORV32_MEM_SRC="$(NEORV32_MEM_SRC)" \
 	  run
 
+nexys_a7:
```

Comment:
I would use `NexysA7` or `Nexys-A7` for consistency.

Reply:
Sure, thats trivial to change.